tests/integration/haproxy_test.go

```diff
@@ -1,18 +1,20 @@
+//go:build integration
 // +build integration
```

Comment:
Can `+build integration` (the older format) be removed?

Reply:
Done in [#37999bb](https://github.com/newrelic/nri-haproxy/pull/157/commits/37999bb8b134681c3c764d25bac09a11b8b629c9)